tensorboard/backend/application.py

```diff
@@ -285,6 +316,7 @@ def __init__(self, plugins, path_prefix="", data_provider=None):
         self._plugins = plugins
         self._path_prefix = path_prefix
         self._data_provider = data_provider
+        self._experimental_plugins = experimental_plugins
```

Comment:
Since this is only used for membership tests, best to convert it to a
`frozenset` for faster queries and more idiomatic set operations (see
suggestion below).